docs/FeatureManagement/FeatureFlagv2.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v2.0.0/FeatureFlag.json",
+    "type": "object",
+    "title": "An Azure App Configuration Feature Declaration",
+    "required": [
+      "id",
+      "enabled",
+      "conditions"
```

Comment:
I understand this is from the old schema, but I don't think `conditions` is required. Can we remove it?